webapp/src/main/webapp/news/components/NewsAppItem.vue

```diff
@@ -85,7 +85,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
               class="newsTime caption" />
           </span>
         </div>
-        <div class="newsViews " v-if="!news.draft && !news.scheduled">
+        <div class="newsViews " v-if="!news.scheduled">
```

Comment:
You should test this modification for a draft article not already posted, normally these informations should not be displayed in that case. (otherwise to be verified with the PO)